action.yml

```diff
@@ -25,9 +25,13 @@ inputs:
     required: false
     default: true
   createGithubReleases:
-    description: "A boolean value to indicate whether to create Github releases after `publish` or not"
+    description: "A value to indicate whether to create Github releases after `publish` or not, and on which mode"
     required: false
-    default: true
+    default: 'true'
+  githubReleaseName:
```

Comment:
It would be great to also add `githubReleaseName` on the docs below the  `createGithubReleases`.